docs/API.md

```diff
@@ -424,64 +425,84 @@ All return a new stamp.
 * stampit.deepPropertyDescriptors()
 * stampit.compose()
 
-### stampit.isStamp(obj)
+
+## Utility functions
+
+### stampit/compose
+
+```js
+import compose from 'stampit/compose'; // or
+const compose = require('stampit/compose');
+```
+
+It's a pure clean standard `compose` function implementation. See [specification](https://github.com/stampit-org/stamp-specification).
+
+### stampit/isStamp
+
+```js
+import isStamp from 'stampit/isStamp'; // or
+const isStamp = require('stampit/isStamp');
+```
 
 Take an object and return `true` if it's a stamp, `false` otherwise.
 
-### stampit.isComposable(obj)
+### stampit/isComposable
 
-Take an object and return `true` if it's a stamp or a stamp descriptor.
+```js
+import isComposable from 'stampit/isComposable'; // or
+const isComposable = require('stampit/isComposable');
+```
 
+Take an object and return `true` if it's a stamp or a stamp descriptor.
 
-## More Examples
 
 ## Chaining methods
 
-Chaining stamps *always* creates new stamps.
+All the methods *always* create new stamps.
```

Comment:
"All these methods *always* creates new stamps." ... not sure why you've removed "s", but that was correct actually ;)